samples/system-test/automlVision.test.js

```diff
@@ -0,0 +1,125 @@
+    const dataSetId = output.split(`\n`)[1].split(`:`)[1].trim();
+    t.true(output.includes(`Dataset display name:  ${dummyDataSet}`));
+
+  // Import Data
+  output = await tools.runAsync(`${cmdDataset} importData -i "${dataSetId}" -p "gs://nodejs-docs-samples-vcm/flowerTraindata20lines.csv"`);
+  t.true(output.includes(`Data imported.`));
+
+  // Check to make sure model doesn't already exist
+  output = await tools.runAsync(`${cmdModel} listModels`);
+  t.false(output.includes(`${testModelName}`));
+
+  // begin training dataset, getting operation ID for next operation 
+  output = await tools.runAsync(`${cmdModel} createModel -i "${dataSetId}" -m "${testModelName}" -t "2"`);
+  const operationName = output.split(`\n`)[0].split(`:`)[1].trim();
+  t.true(output.includes(`Training started...`));
+
+  // poll operation status, here confirming that operation is not complete yet
+  output = await tools.runAsync(`${cmdModel} getOperationStatus -i "${dataSetId}" -o "${operationName}"`);
+  t.true(output.includes(`done: false`));
+});
+
+test(`It should display evaluation from prexisting model`, async t => {
+  const flowersModelId = `ICN723541179344731436`;
+  const flowersDatasetId = `ICN7628761482635807401`;
+  const flowersname = `projects/203278707824/locations/us-central1/datasets/ICN7628761482635807401`;
+  const flowersDisplayName = `flowersTest`;
+  const donotdeleteModelId = `ICN723541179344731436`;
+  
+  // Confirm dataset exists
+  let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+    t.true(output.includes(flowersDisplayName));
+
+  // List model evaluations, confirm model exists
+  output = await tools.runAsync(`${cmdModel} listModelEvaluations -a "${flowersModelId}"`);
+  const flowersEvaluationName = output.split(`/n`)[0].split(`:`)[1].trim();
+
+  // Display evaluation
+  output = await tools.runAsync(`${cmdModel} displayEvaluation -a "${flowersModelId}"`);
+  t.true(output.includes(`Model Precision`));  
+});
+
+test(`It should run Prediction from prexisting model`, async t => {
+  const donotdeleteModelId = `ICN723541179344731436`;
+  const flowersDisplayName = `flowers`;
+
+    // Confirm dataset exists
+    let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+    t.true(output.includes(flowersDisplayName));
+
+    // List model evaluations, confirm model exists
+    output = await tools.runAsync(`${cmdModel} listModelEvaluations -a "${donotdeleteModelId}"`);
+    const flowersEvaluationName = output.split(`/n`)[0].split(`:`)[1].trim();
+
+    // Run prediction on 'testImage.jpg' in resources folder
+    output = await tools.runAsync(`${cmdPredict} predict -i "${donotdeleteModelId}" -f "${sampleImage2}" -s "0.5"`);
+    t.true(output.includes(`dandelion`));
+})
+
+// List datasets
+test(`should list datasets`, async t => {
+  const output = await tools.runAsync(`${cmdDataset} listDatasets`);
+  t.true(output.includes(`List of datasets:`));
+});
```

Comment:
Please add a newline!

Reply:
quick question: where to add newline here?

Reply:
At the end of the file. (I believe opening a file in `vim` and saving it will do this for you.)